docs/feature_macros.md

```diff
@@ -228,6 +228,10 @@ This sends the `<kc>` keydown event to the computer. Some examples would be `KC_
 
 Parallel to `register_code` function, this sends the `<kc>` keyup event to the computer. If you don't use this, the key will be held down until it's sent.
 
+### `tap_code(<kc>);`
+
+This will sent `register_code(<kc>)` and then `unregister_code(<kc>)`.
```

Comment:
Typo in "sent". Might also want to add that it's a useful helper function, something along the lines of:

    This will call `register_code(<kc>)` and then `unregister_code(<kc>)`. Useful when you want to press and release (i.e. tap) a key. Makes for shorter, clearer code compared to calling the two functions yourself.

Reply:
Apparently, I'm unable to proofread my own code. 

But that's why somebody else should always review it. :)